### PR TITLE
[Storage] Bumped version + Pipeline fix

### PR DESCRIPTION
**Pull request by annatisch**
Also includes a few tiny touch-ups as I review the master PR

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by rakshith91** [comment]
some merge conflicts may be? looks like the batching changes are all here

**Comment by annatisch** [comment]
@rakshith91 - that's because the Batching PR didn't sync those changes across the base clients in Files and Queues - which I just copied over.

File: sdk/storage/azure-storage-queue/azure/storage/queue/_version.py [hunk 1]
```diff
@@ -9,4 +9,4 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "12.0.0b4"
+VERSION = "12.0.0b5"
```
Comment (rakshith91) [inline sdk/storage/azure-storage-queue/azure/storage/queue/_version.py]: should this be 0b5 or 12.0.0?

**Pull request merged**